### PR TITLE
privoxy: adjust version for apk

### PR DESCRIPTION
**Pull request by peterwillcn**
Adjust privoxy version for apk by removing the "-stable"
from the OpenWrt version, although it is still needed for
upstream download archive name.

### CODE REVIEW

File: net/privoxy/Makefile [hunk 1]
```diff
@@ -123,7 +124,7 @@ define Package/privoxy/preinst
 
 # stop service if PKG_UPGRADE
 [ "$${PKG_UPGRADE}" = "1" ] && /etc/init.d/privoxy stop >/dev/null 2>&1
-	
+
```
Comment (robimarko) [inline net/privoxy/Makefile:127]: Unrelated change

File: net/privoxy/Makefile [hunk 2]
```diff
@@ -71,9 +72,9 @@ define Package/privoxy/config
 	help
 		Privoxy is a web proxy with advanced filtering capabilities for protecting
 		privacy, modifying web page content, managing cookies, controlling access,
-		and removing ads, banners, pop-ups and other obnoxious Internet junk. 
+		and removing ads, banners, pop-ups and other obnoxious Internet junk.
 		Privoxy has a very flexible configuration and can be customized to suit
-		individual needs and tastes. 
+		individual needs and tastes.
```
[the remark this answers is not in the document]
Comment (peterwillcn) [reply net/privoxy/Makefile:77]: Removed unnecessary spaces and processed them together

File: net/privoxy/Makefile (hunk 1 above)
Comment (peterwillcn) [reply net/privoxy/Makefile:127]: Removed unnecessary spaces and processed them together
